docs/modules/ROOT/assets/images/fire_product_registration.md

```diff
@@ -0,0 +1,30 @@
+# Fire product registration
+
+```mermaid
+sequenceDiagram
+    actor PO as ProductOwner
+    actor IO as InstanceOperator
+    participant G as GIF
+    participant P as FireProduct
+    participant F as PolicyFlow
+    participant R as FireRiskpool
+    participant T as ERC20Token
+    
+    IO ->> G: grant role to ProductOwner
+    PO ->>+ P: deploy FireProduct contract
+    PO ->>+ G: propose FireProduct
+    G -->>- P: after propose callback
+    IO ->>+ G: approve FireProduct
+    G ->> P: get PolicyFlow
+    G ->> G: sets PolicyFlow for Product 
+    F -->> P: GIF assigns PolicyFlow to FireProduct 
+    G ->> P: get Riskpool
+    G ->> G: sets Riskpool for Product
+    R -->> P: GIF assigns FireRiskpool to FireProduct
```

Comment:
There is not function call between FireRiskpool and FireProduct. Please remove this.